docs/overview/office-add-in-availability.md

```diff
@@ -1,14 +1,14 @@
 ---
 title: Office Add-in host and platform availability
 description: Supported requirement sets for Excel, Word, Outlook, PowerPoint, and OneNote.
-ms.date: 10/03/2018
+ms.date: 11/07/2018
 ---
 
 # Office Add-in host and platform availability
 
 To work as expected, your Office Add-in might depend on a specific Office host, a requirement set, an API member, or a version of the API. The following tables contain the available platform, extension points, API requirement sets, and common API requirement sets that are currently supported for each Office application.
 
-If a table cell contains an asterisk ( * ), that means we’re working on it. For requirement sets for Project or Access, see [Office common requirement sets](https://docs.microsoft.com/office/dev/add-ins/reference/requirement-sets/office-add-in-requirement-sets).  
+If a table cell contains an asterisk ( * ), that means we’re working on it.
```

Comment:
Maybe I missed it, but I didn't see any `*`s in the table. Should we remove this note?